src/cargo/util/toml/mod.rs

```diff
@@ -209,7 +209,7 @@ fn convert_toml(
             }
         }
     }
-    return if manifest.project.is_some() || manifest.package.is_some() {
+    return if manifest.package().is_some() {
```

Comment:
I believe both are checked here as we technically still support `[project]`

Reply:
Yes, that is the intention of this function, along with `dev_dependencies()`, `build_dependencies()`, etc

Reply:
I somehow missed the you were calling the function. disregard this comment